hw5 - weatherApp/JS/index.js

```diff
@@ -0,0 +1,129 @@
+let submitFormBtn = document.getElementById('submitFormBtn');
+let cityInput  = document.getElementById('city');
+let countryInput = document.getElementById('country');
+let form = document.forms.cityCountryInputs;
+let block = document.getElementById('block');
+
+let locationHTML = document.getElementById('location');
+let dateHTML = document.getElementById('date');
+let temperatureNumber = document.getElementById('temperatureNumber');
+let weatherDescription = document.getElementById('weatherDescription');
+let temperature = document.getElementById('temperature');
+let humidity = document.getElementById('humidity');
+let pressure = document.getElementById('pressure');
+let cloudCover = document.getElementById('cloudCover');
+let uvIndex = document.getElementById('uvIndex');
+let visibility = document.getElementById('visibility');
+let windSpeed = document.getElementById('windSpeed');
+let windDirection = document.getElementById('windDirection');
+let windDegree = document.getElementById('windDegree');
+
+let temperatureLocationBlock = document.getElementById('temperatureLocationBlock');
+let weatherDescriptionBlock = document.getElementById('weatherDescriptionBlock');
+let weatherInfo = document.getElementById('weatherInfo');
+let welcomePhrase = document.getElementById('welcomePhrase');
+
+let error = document.getElementById('error');
+
+let host = 'http://api.weatherstack.com/current?access_key=b27bbe6126910fbe5e44ee53a22f3937&query=';
+
+function displayWeather() {
+    fetch(host + cityInput.value + ',' + countryInput.value)
```

Comment:
Template strings are much better in such cases:
```suggestion
    fetch(`${host}${cityInput.value},${countryInput.value}`)
```